t/18-qemu.t

```diff
@@ -412,8 +412,9 @@ subtest 'qemu was killed due to the system being out of memory' => sub {
     $mock_proc->redefine(check_qemu_oom => sub { return 0; });
     $proc = qemu_proc('-foo', \%vars);
     $proc->exec_qemu;
-    $proc->_process->stop;
+    $proc->_process->wait_stop;
```

Comment:
This is basically a while loop with `sleep 1` where `1` comes from `$proc`s `sleeptime_during_kill` attribute. Maybe it makes sense to reduce `sleeptime_during_kill` (e.g. to `0.1`).

Reply:
Seems reasonable.